frontend/public/components/utils/resource-log.tsx

```diff
@@ -89,8 +89,10 @@ export const LogControls: React.FC<LogControlsProps> = ({
   containerName,
   podLogLinks,
   namespaceUID,
+  toggleWrapLines,
 }) => {
   const { t } = useTranslation();
+  const [wrapLines, setWrapLines] = React.useState(false);
```

Comment:
We might make this a user setting with `useUserSettings` so it's remembered next time you visit the logs tab. (It also makes the default less important.) @andybraren @christianvogt opinion?

Here's an example: https://github.com/openshift/console/blob/master/frontend/public/components/app-contents.tsx#L64-L67

Reply:
Good point about persisting the choice. In the ticket it indicates that there is possible a log toolbar redesign coming and that this fix should be kept super simple.  Wondering how simple we want to keep this.

Reply:
Agreed about saving this as a local preference. The log toolbar redesign is still in progress, but when it exists we'll still need to persist that preference somewhere.

I think this checkbox + persisting the preference is the level of simplicity we can target for this near-term PR. 🙂

Reply:
@kdoberst I think it's a simple change since we have the `useUserSettings` hook. You should be able to do something like... (untested)

```suggestion
  const [wrapLines, setWrapLines] = useUserSettings<boolean>(
    `${USERSETTINGS_PREFIX}.log.wrapLines`,
    false,
  );
```

Reply:
Implemented